<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>513</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Задание</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Размер прыжка:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Вперед:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="BaseA" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximum" >
           <number>999</number>
          </property>
          <property name="value" >
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Назад:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="BaseB" >
          <property name="maximum" >
           <number>999</number>
          </property>
          <property name="value" >
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="taskNeeded" >
     <property name="text" >
      <string>Задание</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Старт:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="StartPos" >
            <property name="minimumSize" >
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum" >
             <number>-999</number>
            </property>
            <property name="maximum" >
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Флажки</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QSpinBox" name="NeedA" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="minimumSize" >
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="frame" >
             <bool>true</bool>
            </property>
            <property name="minimum" >
             <number>-999</number>
            </property>
            <property name="maximum" >
             <number>999</number>
            </property>
            <property name="value" >
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton" >
            <property name="text" >
             <string>Добавить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QListWidget" name="listWidget" />
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Убрать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="Borders" >
          <property name="text" >
           <string>Границы</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>30</height>
             </size>
            </property>
            <property name="text" >
             <string>От:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>-999</number>
            </property>
            <property name="maximum" >
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="maximumSize" >
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>До:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_2" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>-998</number>
            </property>
            <property name="maximum" >
             <number>999</number>
            </property>
            <property name="value" >
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>9</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder>buttonBox</zorder>
  <zorder>frame_3</zorder>
  <zorder>taskNeeded</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>taskNeeded</sender>
   <signal>clicked(bool)</signal>
   <receiver>frame_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Borders</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel" >
     <x>68</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Borders</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>117</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel" >
     <x>167</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
